src/controllers/StaffController.js

```diff
@@ -2,9 +2,10 @@
 /* eslint-disable consistent-return */
 /* eslint-disable import/no-extraneous-dependencies */
 // bcrypt for password hashing
+const mongoose = require('mongoose');
 const bcrypt = require('bcrypt');
 // import the staff model
-const { enums } = require('../models/base');
+const { enums, privileges, ObjectId } = require('../models/base');
```

Comment:
As discussed previously, all the permissions/validations for CRUD operations are done in the custom methods for the `staff` method. Under no circumstance would you need to use this `privileges` object here as that would result in an unnecessary duplication of code since the called method would always carry out the same check anyway, or even lead to security violations. Care has been taken to properly implement security checks and limits in the models themselves through the defined custom methods, so please always make use of those, as such checks never need to be carried out in the controllers.

If a method doesn't exist for a MongoDB CRUD operation, it needs to first be implemented properly in one of the models, then simply called here as needed, remember?